### ISSUE TITLE
Adding section to Wiki/documentation about how to get AltGr-working

### ISSUE THREAD

**Issue by dd1982** [debauchee/barrier#630]
Adding this trick helped me finally get barrier to a useable state on a Ubuntu-server with a Windows 10-client - with my keyboard layout the following keys are accessible via AltGr: @${}[]\~| so you can imagine my frustration trying to write code until I figured this out.

By saving the config to a file, adding the bold lines below _**and**_ starting barrier with _barrier --config /path/to/config.file_ I finally got AltGr working. 

```
section: screens
    linux:
    windows7:
        meta = altgr    <----
        altgr = shift     <----
end
```

Props to @wjtk4444 that spelled it out in #438- I was just to thick to get it to work.

**Comment by wjtk4444** [comment]
Well, it's just a hack that happens to work for some reason. Windows still sees the Alt key in a weird way, for example PrintScreen is getting captured as Alt+PrintScreen (and some other mild annoyances).

I also can't get Ctrl+Alt+Delete to work, and it's a surprisingly essential shortcut when a fullscreen application stops responding...

**Comment by dd1982** [comment]
Yes - I'm running the server on Ubuntu 19.10 and the client on Win 10 Enterprise 1809 and so far so good.

I have a vague recall of Windows using ctrl-alt-del as a special combo that is not supposed to be interceptable by running software - so that might be a tough one. Even when using MS Remote Desktop ctrl-alt-del is intercepted by the local machine rather than the one you're interacting with.

If any dev wants to try and make a proper fix for AltGr on Windows - the reason people are having trouble can be found in the links below - in Windows pressing AltGr results in funny keycodes;

https://docs.microsoft.com/en-us/windows/win32/inputdev/wm-syskeyup?redirectedfrom=MSDN
https://stackoverflow.com/questions/12252651/right-alt-sends-two-virtual-key-codes

**Comment by mclang** [comment]
I think it would be good to add this also:
https://github.com/debauchee/barrier/issues/134
